Arm-Subsytem/rover_arm_system.hpp

```diff
@@ -216,10 +190,9 @@ class RoverArmSystem : public sjsu::common::RoverSystem
                              pinky, pitch, roll);
   }
 
-  void HomeArm()
+  void HomeArmSystem()
   {
-    HomeShoulder();
-    HomeElbow();
+    // HomeArm();
     hand_.HomeHand(mc_data_.arm_speed, rotunda_.GetOffsetAngle());
   }
```

Comment:
Need to call the arm_.HomeShoulder() & arm_.HomeElbow functions here most likely